### PR TITLE
Add handling CC and BCC to Sender

### PR DESCRIPTION
**Pull request by marcinwarzybok**
| Q               | A
| --------------- | -----
| Branch?         | master
| Bug fix?        | no
| New feature?    | yes
| BC breaks?      | no
| Deprecations?   | no
| Related tickets | -
| License         | MIT

This is my first contribution so please forgive me any missing parts/faults

### CODE REVIEW

**Comment by Roshyo** [comment]
Actually is does break BC, since the interface changed.

**Comment by marcinwarzybok** [comment]
> Actually is does break BC, since the interface changed.

even if it extends the interface for not required fields (so default they will be an empty array) ?

**Comment by Roshyo** [comment]
Yeah, you still have to implement those fields in classes.

**Comment by marcinwarzybok** [comment]
hah that's right. I completely forgot that other than SwiftMailer implementations ;/

File: src/Component/Sender/Adapter/AdapterInterface.php [hunk 1]
```diff
@@ -26,6 +26,8 @@ public function send(
         EmailInterface $email,
         array $data,
         array $attachments = [],
-        array $replyTo = []
+        array $replyTo = [],
+        array $ccRecipients = [],
```
Comment (lchrusciel) [inline src/Component/Sender/Adapter/AdapterInterface.php]: Let's add it to the new interface, which will extend this one. Then keep existing logic and propagate with additional data if `cc` of `bcc` recipients are part of the contract.

**Comment by lchrusciel** [comment]
Just to pin ref. https://github.com/Sylius/SyliusMailerBundle/issues/5

**Comment by Healkiss** [comment]
There is more than 2 years that this bundle should be deprecated but always in use.
This feature is usefull and valide. Could we merge it maybe ?

**Comment by Roshyo** [comment]
lchrusciel's comment is still valid though. It should be added too the new Interface instead of modifying the existing one.

**Comment by vvasiloi** [comment]
There's already a PR to deprecate the Swiftmailer integration in favor of Symfony Mailer: https://github.com/Sylius/SyliusMailerBundle/pull/102

**Pull request closed without merging**